src/ctia/stores/es/mapping.clj

```diff
@@ -118,6 +118,11 @@
    {:type token
     :value all_token}})
 
+(def assertion
+  {:properties
+   {:name all_text
+    :value all_text}})
```

Comment:
when you use the mapping `all_text`in CTIA it enables 2 things:
1. search on _all fields meaning that you can search without specifying the field that it should match.
2. search on words in the corresponding text.

As a consequence, for the name `cisco:ctr:group`from that [example](https://github.com/threatgrid/ctim/blob/master/src/ctim/examples/identity_assertions.cljc#L31), the following queries should match that text: 
* assertions.name:"cisco:ctr:group"
* assertions.name:"cisco ctr"
* "cisco:ctr:group"
* "cisco ctr"
* ctr
* cisco
* group
note that `cisco:ctr:group`will generate an error because it's a reserved word for lucene queries.

If you use `all_token`, it should only match:
* assertions.name:"cisco:ctr:group"
* "cisco:ctr:group"

If you use the versions without the `all_`prefix it will force to specify the searched field: `assertions.name:...`

Thus, IMHO we should have:
``` clojure
(def assertion
  {:properties
   {:name all_token
    :value all_text}})
```